dnd-bot/dnd_bot/logic/game/initialize_world.py

```diff
@@ -177,31 +143,35 @@ def spawn_players_positions(spawning_points, num_players):
         return players_positions
 
     @staticmethod
-    def add_entity(entity_row, entity_class, x, y, game_token, game_id):
+    def add_entity(entity_row, entity_name, x, y, game_token, game_id, enemies_json, map_elements_json):
```

Comment:
`game_id`, `enemies_json` oraz `map_elements_json` są nieudokumentowane, jeżeli już używamy tych docstringów to fajnie by było je dodać nawet bez opisu